### PR TITLE
Move HTML job posting pages to markdown

### PR DESCRIPTION
**Pull request by tvolkert**

### CODE REVIEW

File: src/jobs/windows.md
Comment (sfshaza2) [inline src/jobs/windows.md:8]:
```suggestion
Flutter is the most popular multi-platform UI toolkit,
with shipped support for iOS, Android, Web, Windows, macOS,
and Linux. On Windows, we're allowing Flutter developers to target
an additional 1B users, and we're offering a better alternative to
desktop developers who use Electron, Windows Forms, WPF, or (shudder) UWP!
```

File: src/jobs/windows.md
Comment (sfshaza2) [inline src/jobs/windows.md:10]:
```suggestion
The Flutter Desktop Team is responsible for the desktop-specific
(Windows, macOS, Linux) embeddings of the Flutter engine,
which includes enabling platform plugins, nesting platform views
inside Flutter apps, and wiring-up accessibility hooks on those
platforms&mdash;and anything else that helps Flutter feel “native” when run in those environments.
```

File: src/jobs/windows.md
Comment (sfshaza2) [inline src/jobs/windows.md:14]:
```suggestion
In the "Windows Engineer" role you'll be part of the team that makes
Flutter apps the absolute best they can be when running on Windows, including:
```

File: src/jobs/windows.md
Comment (sfshaza2) [inline src/jobs/windows.md:18]:
```suggestion
*   Adding support for shell / window drag and drop.
```

File: src/jobs/windows.md
Comment (sfshaza2) [inline src/jobs/windows.md:21]:
```suggestion
*   Nurturing a thriving community of contributions from open-source
    developers by reviewing GitHub issues and pull requests (PRs).
```

File: src/jobs/windows.md
Comment (sfshaza2) [inline src/jobs/windows.md:27]:
```suggestion
Upon joining Google, you will be paired with a formal mentor,
who will help guide you in the process of ramping up, forging relationships,
and learning the systems you’ll need to do your job. Your manager can also
help you find mentors who can coach you as you navigate your career at Google.
In addition to formal mentors, we work and train together so that we are
always learning from one another, and we celebrate and support the career progression
of our team members.
```

File: src/jobs/windows.md
Comment (sfshaza2) [inline src/jobs/windows.md:31]:
```suggestion
Here on the Flutter team and at Google,
we embrace our differences and are
[committed to furthering our culture of inclusion](https://flutter.dev/culture).
In addition to groups like the [Flutteristas](https://flutteristas.org/),
[Employee Resource Groups (ERGs)](https://diversity.google/commitments/)
are employee-initiated networks for supporting underrepresented employees
and their allies with shared values of creating belonging across their communities and Google.
```

File: src/jobs/windows.md
Comment (sfshaza2) [inline src/jobs/windows.md:35]:
```suggestion
Our team also puts a high value on work-life balance.
Striking a healthy balance between your personal and professional life
is crucial to your happiness and success here, which is why we aren’t
focused on how many hours you spend at work or online. Instead,
we’re happy to offer a flexible schedule so you can have a more productive
and well-balanced life&mdash;both in and outside of work.
```

File: src/jobs/windows.md
Comment (sfshaza2) [inline src/jobs/windows.md:39]:
```suggestion
Any Google office location, or fully remote in a timezone 3 hours
behind or ahead of Pacific Standard Time (US West Coast).
```

File: src/jobs/windows.md
Comment (sfshaza2) [inline src/jobs/windows.md:47]:
```suggestion
*   Streamline Flutter’s mechanism for embedding native Windows UI views
    or communicating with native Windows libraries.
```

File: src/jobs/windows.md
Comment (sfshaza2) [inline src/jobs/windows.md:49]:
```suggestion
### Traits that will help you in this role
```

**Review by sfshaza2 — APPROVED** [review]
LGTM, once the suggested edits land.

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**
